code/modules/mob/camera/imaginary_friend.dm

```diff
@@ -82,6 +88,51 @@
 
 	return ..()
 
+/mob/camera/imaginary_friend/verb/toggle_darkness()
+	set category = "Imaginary Friend"
+	set name = "Toggle Darkness"
+
+	switch(lighting_alpha)
+		if(LIGHTING_PLANE_ALPHA_VISIBLE)
+			lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE
+		if(LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE)
+			lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
+		if(LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE)
+			lighting_alpha = LIGHTING_PLANE_ALPHA_INVISIBLE
+		else
+			lighting_alpha = LIGHTING_PLANE_ALPHA_VISIBLE
+
+	update_sight()
```

Comment:
thats repeated from somewhere else right ? if so, turn that into a proc at mob and just add them to their verbs list
verbs += /mob/proc/blabla
something like that

Reply:
Nvm this, spoke to wayland and the set category is a problem